source/containers.tex

```diff
@@ -2203,10 +2228,16 @@
 \pnum
 The member function templates
 \tcode{find}, \tcode{count}, \tcode{contains},
-\tcode{lower_bound}, \tcode{upper_bound}, and \tcode{equal_range}
+\tcode{lower_bound}, \tcode{upper_bound}, \tcode{equal_range},
+\tcode{erase}, and \tcode{extract}
 shall not participate in overload resolution unless
 the \grammarterm{qualified-id} \tcode{Compare::is_transparent} is valid
 and denotes a type\iref{temp.deduct}.
+Additionally, the member function templates \tcode{erase} and \tcode{extract}
```

Comment:
"erase and extract" or "extract and erase"? Please be consistent (see comment below).

Reply:
The synopsis shows extract before erase, so this one needs fixing.